src/field_theory/adjoin.lean

```diff
@@ -82,14 +82,27 @@ by { ext, rw [mem_to_subalgebra, algebra.mem_bot, mem_bot] }
 @[simp] lemma top_to_subalgebra : (⊤ : intermediate_field F E).to_subalgebra = ⊤ :=
 by { ext, rw [mem_to_subalgebra, iff_true_right algebra.mem_top], exact mem_top }
 
-/--  Construct an algebra isomorphism from an equality of subalgebras -/
-def subalgebra.equiv_of_eq {X Y : subalgebra F E} (h : X = Y) : X ≃ₐ[F] Y :=
+/--  Construct an algebra isomorphism from an equality of intermediate fields -/
+@[simps]
```

Comment:
```suggestion
@[simps apply]
```
else I think the linter will complain about the generated `equiv_of_eq_symm_apply` being provable by simp